ddl/index.go

```diff
@@ -815,59 +808,45 @@ func (r *reorgIndexTask) String() string {
 	return "physicalTableID" + strconv.FormatInt(r.physicalTableID, 10) + "_" + "[" + r.startHandle.String() + "," + r.endHandle.String() + rightParenthesis
 }
 
-type addIndexResult struct {
+type backfillResult struct {
 	addedCount int
 	scanCount  int
 	nextHandle kv.Handle
 	err        error
 }
 
-// addIndexTaskContext is the context of the batch adding indices.
-// After finishing the batch adding indices, result in addIndexTaskContext will be merged into addIndexResult.
-type addIndexTaskContext struct {
+// backfillTaskContext is the context of the batch adding indices.
+// After finishing the batch adding indices, result in backfillTaskContext will be merged into backfillResult.
```

Comment:
The comments seem to be outdated. (`adding indices`)

Reply:
Processing this TODO will add more than 500 lines of modification, which is not conducive to review. I mentioned this question yesterday.